### DISCUSSION TITLE
Обход блокировки discord чат + войс.

### DISCUSSION THREAD

**Discussion by SpiritOTHawk** [dovecoteescapee/ByeDPIAndroid#173] [Show and tell]
![375283594-68ddd137-3fd6-4c65-9541-cf7d336d11fa](https://github.com/user-attachments/assets/0d1089f9-3621-40ce-9095-b26c8541bb7e)
![375283556-780459ca-6847-487b-94eb-9ad8d912d600](https://github.com/user-attachments/assets/fda415c1-2a80-4a8f-8b25-9563445410ea)

**Comment by ladccsFS** [comment]
Доброе, дс(тч+гч) грузит, а вот с ютубом проблемы, от 20 сек. надо ждать чтобы ролик начал воспроизводиться, но может быть ток у меня такая проблема. (Ростелеком)
Спасибо за настройки.

**Comment by cgvalyok1** [comment]
Доброго времени суток всем страждущим. Пробовал эти настройки с мобильного инета (теле2) и вайфая (подряд). Дискорд работает в обоих случаях (и текст и голос). Ютуб с вайфая не завелся, но с мобилки вроде сработал. Приложуха даже починила инсту

**Comment by Gruit13** [comment]
Не нахожу всё это в настройках

**Reply by cgvalyok1** [reply]
В секции ByeDPI убрать тумблер с  use command line settings и станет доступна кнопка UI editor. В ней все настройки со скрина

**Comment by Dronatar** [comment]
Много лишнего, нужно лишь:
В Protocols включи Desync UDP
В UDP сделай UDP fake count 1

**Comment by ilya7even** [comment]
работал с данными настройками, сегодня перестал. йота.
